### DISCUSSION TITLE
Multiple streamlisteners possible?

### DISCUSSION THREAD

**Discussion by aneesh3397** [tweepy/tweepy#1374] [Questions] [answered]
Hello! I was wondering if I could get some clarification on limits on the number of open streams you can have with a single account. 

What I've tried:

Made a simple streamlistener based on the tutorial here: http://docs.tweepy.org/en/latest/streaming_how_to.html. I then tried running 2 instances of that script simultaneously. It seems that I can run two scripts with no trouble but starting three or more leads to unpredictable behavior. Sometimes the new script runs, sometimes it shuts down with no exceptions raised. 

I've come across posts saying that you are only allowed to have one open stream at a time, and starting a new stream causes the old one to be killed. This is not the behavior I am seeing. Haven't been able to find anything in the official docs regarding this topic, would appreciate some clarification.

Thanks!

**Comment by Harmon758** [comment, marked as answer]
Twitter's documentation for statuses/filter says:
> One filter rule on one allowed connection, disconnection required to adjust rule

https://developer.twitter.com/en/docs/tweets/filter-realtime/overview

In practice, there might be some leeway for two or three simultaneous streams so that you can have redundancy in case a stream unexpectedly disconnects, but that is the documented limit.
